vllm/entrypoints/openai/api_server.py

```diff
@@ -253,6 +253,15 @@ def _cleanup_ipc_path():
             multiprocess.mark_process_dead(engine_process.pid)
 
 
+async def validate_json_request(raw_request: Request):
+    content_type = raw_request.headers.get("content-type", "").lower()
+    if "application/json" not in content_type:
```

Comment:
can we check for equality here? No need to make it more ambiguous.

Reply:
I had implemented the equality check on content_type